Formula/tcpreplay.rb

```diff
@@ -1,9 +1,9 @@
 class Tcpreplay < Formula
   desc "Replay saved tcpdump files at arbitrary speeds"
   homepage "https://tcpreplay.appneta.com/"
-  url "https://github.com/appneta/tcpreplay/releases/download/v4.3.3/tcpreplay-4.3.3.tar.gz"
-  sha256 "ed2402caa9434ff5c74b2e7b31178c73e7c7c5c4ea1e1d0e2e39a7dc46958fde"
-  license "GPL-3.0"
+  url "https://github.com/appneta/tcpreplay/releases/download/v4.3.4/tcpreplay-4.3.4.tar.gz"
+  sha256 "ee065310806c22e2fd36f014e1ebb331b98a7ec4db958e91c3d9cbda0640d92c"
+  license "GPL-3.0-or-later"
```

Comment:
It's got a few more licenses: https://metadata.ftp-master.debian.org/changelogs//main/t/tcpreplay/tcpreplay_4.3.3-2_copyright

`BSD-4-Clause`, `ISC`, `BSD-2-Clause`, `BSD-3-Clause`

Reply:
included all the licenses.